lib/chef-dk/command/generator_commands/build_cookbook.rb

```diff
@@ -29,11 +29,17 @@ class BuildCookbook < Base
 
         attr_reader :cookbook_name_or_path
 
+        option :pipeline,
+          :long  => "--for PIPELINE",
+          :description => "Use PIPELINE to set target branch to something other than master for the build_cookbook",
+          :default => "master"
+
         options.merge!(SharedGeneratorOptions.options)
 
         def initialize(params)
           @params_valid = true
           @cookbook_name = nil
+          @pipeline = "master"
```

Comment:
Is this required if we've already got the `:default` set above in the options hash?